instrumentation/couchbase/couchbase-2.6/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/couchbase/v2_6/CouchbaseNetworkInstrumentation.java

```diff
@@ -25,16 +22,9 @@
 
 public class CouchbaseNetworkInstrumentation implements TypeInstrumentation {
 
-  @Override
-  public ElementMatcher<ClassLoader> classLoaderOptimization() {
-    return hasClassesNamed("com.couchbase.client.core.endpoint.AbstractGenericHandler");
-  }
-
   @Override
   public ElementMatcher<TypeDescription> typeMatcher() {
-    // Exact class because private fields are used
-    return nameStartsWith("com.couchbase.client.")
-        .and(extendsClass(named("com.couchbase.client.core.endpoint.AbstractGenericHandler")));
+    return named("com.couchbase.client.core.endpoint.AbstractGenericHandler");
```

Comment:
this instrumentation wouldn't work on subclass anyways since accessing private fields of `AbstractGenericHandler`